docs/buildah-build.1.md

```diff
@@ -708,8 +708,10 @@ Skip stages in multi-stage builds which don't affect the target stage. (Default
 
 **--squash**
 
-Squash all of the image's new layers into a single new layer; any preexisting layers
-are not squashed.
+With --squash Buildah squash all of the image’s layers (including those inherited from a base image) into a single new layer.
+
+Without --squash (or with --squash=false) Buildah just squash all of the image's new layers into a single new layer; any preexisting layers are not squashed. This is because Buildah squash the image's new layers by default. But be aware with the combination of --squash=false and --layer it will not squash any layer.
```

Comment:
Thank you for taking this on, @dandan2000. I think I can see what you're trying to say, but unfortunately it does not parse grammatically. And there's no `--layer` option (the name is `--layers`, with an `s`). Would this wording address your concerns instead?

>Squash all layers, including those from base image(s), into one single layer. (Default is `false`).
>
>By default, Buildah preserves existing base-image layers and adds only one new layer on a build.
>The `--layers` option can be used to preserve intermediate build layers.